cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactStore.java

```diff
@@ -189,9 +193,10 @@
   private final DatasetFramework datasetFramework;
   private final Transactional transactional;
   private final Impersonator impersonator;
+  private final Set<String> excludedRequirements;
 
   @Inject
-  ArtifactStore(DatasetFramework datasetFramework,
+  ArtifactStore(CConfiguration configuration, DatasetFramework datasetFramework,
```

Comment:
Call the `Configuration` `cConf` to be consistent with other places.

Reply:
Done